packages/quick_blue/darwin/QuickBlueDarwin.swift

```diff
@@ -220,6 +221,11 @@ extension QuickBlueDarwin: FlutterStreamHandler {
     print("QuickBlueDarwin onListenWithArguments: \(name)")
     if name == "scanResult" {
       scanResultSink = events
+    } else if name == "availabilityChange" {
+      availabilityChangeSink = events
+      availabilityChangeSink?([
```

Comment:
We'd better leave a comment here `// FIXME https://github.com/woodemi/quick.flutter/issues/42`

Reply:
It is fixed already. Why do we need the comment?

Reply:
We usually dont fire an event as soon as `onListen`. A `// HACK` may indicate why the code is here

Reply:
It is not really a hack. It actually serves a double purpose:
- Initializes CBCentralManager
- Returns state when hot restarting otherwise CBCentralManager is already initialized and no state is returned.

I added a comment to indicate this.